crates/texlab/src/server/options.rs

```diff
@@ -125,7 +126,9 @@ pub struct ExperimentalOptions {
     pub verbatim_environments: Vec<String>,
     pub citation_commands: Vec<String>,
     pub label_definition_commands: Vec<String>,
+    pub label_definition_prefixes: Vec<(String, String)>,
```

Comment:
What do you think about using a `FxHashMap<String, String>` instead of a list of tuples? Probably, this depends a bit on the editor ergonomics.

Reply:
For my neovim setup, this boils down to `{{"asmref", "asm:"}, {"goalref", "goal:"}}` versus `{asmref = "asm:", goalref = "goal:"}`. 
I'm fine with either. However, I just noticed another issue here.

What gets lost is the ability to map one command to multiple different prefixes, which may be useful if someone `\renewcommand`s.
The current PR doesn't respect these multiply defined prefixes anyway, e.g., https://github.com/DasNaCl/texlab/blob/402503cb78243b1f14c7deea59ccb7375faf9474/crates/parser/src/config.rs#L11

Do we want to support this or just stick with one-to-one mappings? For now, I've changed the code to reflect the one-to-one mapping, i.e., I changed the type from `Vec<(String, String)>` to `FxHashMap<String, String>`.